worker/lib/index.ts

```diff
@@ -78,6 +85,21 @@ async function setup(): Promise<express.Application> {
 			}
 		},
 	);
+	app.post(
+		'/dut/setup',
```

Comment:
We weren't doing setup before??
Did you add this or the PR already had this? If it's the latter then it makes sense. We did move to a completely new way of working with DeviceInteractor

Reply:
This is something I forgot to remove! Will get rid of this, I was just experimenting with it